nengo_gui/static/test/datastore_test.js

```diff
@@ -196,4 +196,4 @@ describe("GrowableDataStore", function() {
         assert.deepEqual(data_store.get_last_data(), [5.1, 5.2, 5.3]);
     });
 
-});
+});
```

Comment:
Just a nitpick, but FYI, you can avoid pushing not-really-helpful changes like this by reviewing the code in your commit with `git commit -p`. Terry showed me to do that and the quality of my pull requests greatly improved. (: